utils/jsonapi/manifest/location.go

```diff
@@ -0,0 +1,69 @@
+package manifest
+
+import (
+	"fmt"
+	"path"
+	"runtime"
+)
+
+func getLocation(browser, libpath string, globalInstall bool) (string, error) {
+	switch platform := runtime.GOOS; platform {
+	case "darwin":
+		{
+			switch browser {
+			case "firefox":
+				{
+					if globalInstall {
+						return "/Library/Application Support/Mozilla/NativeMessagingHosts/%s.json", nil
```

Comment:
All these paths should probably be stored in an easily configurable dictionary. Perhaps in a config. file? Especially the linux ones where you cannot make such a strong assumption about the shape of people's filesystems.

Reply:
Storing in a dictionary was actually my first approach, but you end up with at least three nesting levels and dynamic nesting levels (more like a tree), which needs special handling too.

Not sure about the config file: Those are typically written by gopass on initial run/configuration. If we ship a separate file, wouldn't that deviate from the single binary contains it all principle?

One other solution I could think of is making heavy usage of [templating](https://golang.org/pkg/text/template/) but I am not sure if that is more readable at the end.